Firebase/Messaging/FIRMessagingUtilities.m

```diff
@@ -174,6 +174,7 @@ uint64_t FIRMessagingGetFreeDiskSpaceInMB(void) {
   }
 }
 
+#if TARGET_OS_IOS || TARGET_OS_TV
 UIApplication *FIRMessagingUIApplication(void) {
```

Comment:
Just in case you need a shared application, you may use `+[GULAppDelegateSwizzler sharedApplication]` method

Reply:
Should we move this to GULAppEnvironmentUtil?

Reply:
Yeah, it looks like a better place for it. We may move it in a next release

Reply:
I've replaced with GULAppDelegateSwizzler, so it's easy to swap when you are ready to move.